web-app/js/portal/search/FacetMapPanel.js

```diff
@@ -57,6 +58,12 @@ Portal.search.FacetMapPanel = Ext.extend(Portal.common.MapPanel, {
         });
     },
 
+    _initSpatialConstraintValidator: function() {
+        this.geoFacetMapToolbar.spatialConstraintControl.validator = new Portal.filter.validation.SpatialConstraintValidator({
```

Comment:
It is not nice that we have to go into the object and directly set the validator. For the main map we set the validator with dependency injection, when the spatial constraint is created, which is much better. However, there is a confusion of maps here (2 different maps are referenced by `this.map` during the constructor) and I couldn't find how to untangle that.